cognitive/src/main/scala/com/microsoft/azure/synapse/ml/cognitive/anomaly/MultivariateAnomalyDetection.scala

```diff
@@ -524,6 +526,13 @@ class SimpleDetectMultivariateAnomaly(override val uid: String) extends Model[Si
 
   def getModelId: String = $(modelId)
 
+  val diagnosticsInfo = new CognitiveServiceStructParam[DiagnosticsInfo](this, "diagnosticsInfo",
```

Comment:
What is this CognitiveServicesStruct param? Does this provide a general way of letting the python side create wrappers too? If so thats pretty awesome